src/sedcli_main.c

```diff
@@ -104,8 +104,8 @@ static cli_command sedcli_commands[] = {
 	{
 		.name = "discovery",
 		.short_name = 'D',
-		.desc = "Performs SED level0 discovery(works only with NVMe passthru mechanism)",
-		.long_desc = "Performs SED level0 discovery and provides basic info about the device(works only with NVMe passthru mechanism)",
+		.desc = "Performs SED Opal Device discovery(works only with NVMe passthru mechanism)",
+		.long_desc = "Performs SED Opal Device discovery. This command provides level0 discovery info. and tper properties of the device(works only with NVMe passthru mechanism)",
```

Comment:
It would be more extensible if you just say: "Performs discovery of device properties (e.g. through Level 0 and Level 1 Discovery)"